app/src/main/java/fr/free/nrw/commons/Zoom.java

```diff
@@ -0,0 +1,117 @@
+package fr.free.nrw.commons;
+
+import android.app.Activity;
+import android.graphics.Matrix;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import android.graphics.PointF;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.View.OnTouchListener;
+import android.widget.ImageView;
+
+public class Zoom  implements OnTouchListener {
+
+
+
+
+    Matrix matrix = new Matrix();
+    Matrix savedMatrix = new Matrix();
+
+
```

Comment:
We usually have one blank line between logical separations.

Please refer to the google style guide. https://google.github.io/styleguide/javaguide.html

Reply:
will follow the guidelines and make changes accordinly